llvm/lib/Target/RISCV/RISCVInstrGISel.td

```diff
@@ -0,0 +1,26 @@
+//===-- RISCVInstrGISel.td - RISC-V GISel target pseudos ----*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+/// \file
+// RISC-V GlobalISel target pseudo instruction definitions. This is kept
+// separately from the other tablegen files for organizational purposes, but
+// share the same infrastructure.
+//
+//===----------------------------------------------------------------------===//
+
+class RISCVGenericInstruction : GenericInstruction {
+  let Namespace = "RISCV";
+}
+
+// Pseudo equivalent to a RISCVISD::FPCLASS.
```

Comment:
Can you rename RISCVISD::FPCLASS to RISCVISD::FCLASS as pre or post commit?

Reply:
Renamed in https://github.com/llvm/llvm-project/commit/e096732307d1eb9476fc2b88dfa12586250c8111.  I've also updated this patch